### ISSUE TITLE
Further repository setup

### ISSUE THREAD

**Issue by chrisroadmap** [chrisroadmap/climateforcing#22]
- [ ] issue request template
- [ ] pull request template
- [ ] continous integration for deployment
  - [ ] pypi
  - [ ] conda

**Comment by znicholls** [comment]
@chrisroadmap I don't know if it's worth the effort now but you could also put this on the conda-forge channel if you wanted...

**Comment by chrisroadmap** [comment]
Yeah that's one for the to-do. I stuck it on my own channel as I'm using this on a system where you can only conda-install - so was a short-term fix.
